src/main/java/core/basesyntax/service/impl/FileReaderCsvImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderCsvImpl implements FileReader {
+
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(filePath))) {
+            br.readLine();
+
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
```

Comment:
```suggestion
        try () {
            return Files.readAllLines(...);
        } catch (IOException e) {
```

Reply:
Thank you, Files.readAllLines(...); used